### DISCUSSION TITLE
v6 with Framer Motion?

### DISCUSSION THREAD

**Discussion by jstevens79** [styled-components/styled-components#4063] [Q&A] [answered]
Has anyone been able to get v6 working with Framer Motion? I tried adding StyleSheetManager with the emotion types used in prop forwarding mentioned in the documentation, but none of the animations/transitions work after doing so.

**Comment by manelio** [comment]
It should work allowing props to be forwarded:

```
const ListItem = styled(motion.li).withConfig({
  shouldForwardProp: () => true,
})`
margin: 0;
`
```

**Comment by quantizor** [comment, marked as answer]
styled-components v6 works with Framer Motion. As noted in the thread, use `.withConfig` to ensure motion props are forwarded:

```js
const AnimatedItem = styled(motion.li).withConfig({
  shouldForwardProp: () => true,
})`
  margin: 0;
`;
```

Alternatively, with Framer Motion v11+, use `motion.create()`:

```js
const MotionDiv = motion.create(MyStyledComponent);
```

The key is ensuring that Framer Motion animation props (`animate`, `initial`, `exit`, etc.) are not filtered out by styled-components default prop filtering.
